.github/renovate.json

```diff
@@ -1,5 +1,11 @@
 {
-  "extends": ["github>balena-os/renovate-config"],
+  "extends": [
+    "github>balena-os/renovate-config",
+    ":automergeStableNonMajor",
```

Comment:
This preset is good, but does not work with versioning schemes without range support (e.g. Docker and Regex) so in balena-io we are also using [this](https://github.com/balena-io/renovate-config/blob/58e847566a9f048cbccdcc992b236ef131bc0231/default.json#L98-L102).